.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,4 +20,7 @@ Unless your change is trivial or a small documentation fix (e.g., a typo or rewo
   Also make sure to end the sentence with a `.`.
 
 - [ ] Add yourself to `AUTHORS` in alphabetical order.
+
+If you leave the *Allow edits from maintainers* checked, the mainteiners might squash your commits before merging. If unchecked, maintainers
+will probably ask you to squash yourself then.
```

Comment:
This is confusing.
squash-merging is not the same as (force-)pushing to the branch ("allow edits from maintainers").

> might squash your commits before merging

This sounds like it would mean "squashed, force-pushed and then merged" (not squash-merged).

(Squash-merging is only an edit in the sense of maybe changing the commit message(, and having a single commit only of course.).)

So while a sentence like that is good, and the checkbox could be used as an indicator for that, it should be clarified.
I can think of it and make a suggestion in case my point does not come across clearly.. :+1: 

(also there's a typo at least: mainteiners)

Reply:
Changed, thanks. Feel free to make further suggestions directly. 👍